install/installer/pkg/components/usage/configmap.go

```diff
@@ -62,7 +63,12 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		if expUsageConfig.DefaultSpendingLimit != nil {
 			cfg.DefaultSpendingLimit = *expUsageConfig.DefaultSpendingLimit
 		}
-		cfg.CreditsPerMinuteByWorkspaceClass = expUsageConfig.CreditsPerMinuteByWorkspaceClass
+		cfg.CreditsPerMinuteByWorkspaceClass = make(map[string]float64)
```

Comment:
This doesn't need to be inside the `expUsageConfig` because it doensn't relly on the `expUsageConfig` being set. It relies on the `workspaceClassConfig` variable only.